src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/CommandParsers.tt

```diff
@@ -103,7 +103,8 @@ namespace NuGet.CommandLine.XPlat
             app.Command("<#= commandName #>", <#= commandFormalName #>Cmd =>
             {
 <#
-        foreach (var option in command.Descendants()) {
+        foreach (var option in command.Descendants())
```

Comment:
Most of the `var`'s in this file should be replaced with the explicit type.

Reply:
Changed all 3 .tt files... one of them, i ended up cutting a bunch of dead code. thx!